dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ExclusionFilesTest.java

```diff
@@ -340,14 +340,15 @@ public void testWriteExclusionFile_indent()
 
     ExclusionFiles.write(output, linkageErrors);
 
-    String actual = new String(Files.readAllBytes(output));
+    String actual = new String(Files.readAllBytes(output)).replaceAll("\\R", "\n");
 
     String expected =
         new String(
-            Files.readAllBytes(
-                absolutePathOfResource(
-                    "exclusion-sample-rules/expected-exclusion-output-file.xml")),
-            StandardCharsets.UTF_8);
+                Files.readAllBytes(
+                    absolutePathOfResource(
+                        "exclusion-sample-rules/expected-exclusion-output-file.xml")),
+                StandardCharsets.UTF_8)
+            .replaceAll("\\R", "\n");
```

Comment:
Memo: In the new Windows environment, which changed jdk1.8.0_152 to jdk1.8.0_211, the return character was different. This change ignores the difference.